src/commands/node.mjs

```diff
@@ -356,6 +358,50 @@ export class NodeCommand extends BaseCommand {
     }
   }
 
+  uploadPlatformSoftware (ctx, task, localBuildPath) {
```

Comment:
Need to simplify the complexity of `uploadPlatformSoftware` and support the flags:
```
--local-build-path
--app
--app-config
```

Since `--local-build-path` will contain the full path to the JAR files, some if-else logic will become simplified I believe.